_component_design/search.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Search Bar
+title: Search
 parent: Search and Filters
 layout: component
 category: Components
 usage: >
-  A search box is a combination of an input field and a submit button. One may think that the search box doesn’t need a design; after all, it’s just two simple elements. But since the search box is one of the most frequently used design elements on content-heavy websites, its usability is critical.
-preview-image: preview-images/search.svg
-resource: true
+  By using the Helix search pattern, users can see the available facets or filters within the search field and construct their own search criteria.
+preview-image: preview-images/assisted-search.svg
```

Comment:
Was it intentional to use the `assisted-search.svg` image?

Reply:
This article is hidden now